src/pages/components/back-to-top.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Back to top
-description: asdf.
+description: The Back to top component is a fixed button that allows the user to scroll to the top of a long page.
```

Comment:
```suggestion
description: The Back to top component is a fixed button that allows the user to quickly navigate to the top of a page.
```

Reply:
I was hoping you would tweak it. :wink: thanks!